src/VisualStudio/CSharp/Impl/Options/Formatting/CodeStylePage.cs

```diff
@@ -54,22 +54,22 @@ internal static string GetCurrentEditorConfigOptionsString(OptionSet optionSet)
             editorconfig.AppendLine();
             // this. preferences
             editorconfig.AppendLine("# this. preferences");
-            editorconfig.Append("dotnet_style_qualification_for_field = ");
-            editorconfig.AppendLine(CSharpCodeStyleOptions_GenerateEditorconfig(optionSet, CodeStyleOptions.QualifyFieldAccess));
-            editorconfig.Append("dotnet_style_qualification_for_property = ");
-            editorconfig.AppendLine(CSharpCodeStyleOptions_GenerateEditorconfig(optionSet, CodeStyleOptions.QualifyPropertyAccess));
-            editorconfig.Append("dotnet_style_qualification_for_method = ");
-            editorconfig.AppendLine(CSharpCodeStyleOptions_GenerateEditorconfig(optionSet, CodeStyleOptions.QualifyMethodAccess));
-            editorconfig.Append("dotnet_style_qualification_for_event = ");
-            editorconfig.AppendLine(CSharpCodeStyleOptions_GenerateEditorconfig(optionSet, CodeStyleOptions.QualifyPropertyAccess));
+            // dotnet_style_qualification_for_field
+            DotNetCodeStyleOptions_GenerateEditorconfig(optionSet, CodeStyleOptions.QualifyFieldAccess, editorconfig);
+            // dotnet_style_qualification_for_property
+            DotNetCodeStyleOptions_GenerateEditorconfig(optionSet, CodeStyleOptions.QualifyPropertyAccess, editorconfig);
+            // dotnet_style_qualification_for_method
+            DotNetCodeStyleOptions_GenerateEditorconfig(optionSet, CodeStyleOptions.QualifyMethodAccess, editorconfig);
+            // dotnet_style_qualification_for_event
+            DotNetCodeStyleOptions_GenerateEditorconfig(optionSet, CodeStyleOptions.QualifyEventAccess, editorconfig);
```

Comment:
awesome.  this looks much cleaner!   Note: i think the comments are a bit redundant as well.  but i could see leaving htem in :)